tests/UnambiguousStringTest.php

```diff
@@ -50,6 +50,8 @@ public function unambiguousString_VariousCharacterCounts_HasExpectedResult(int $
      */
     public function isUnambiguousString_VariousStrings_HasExpectedResult(string $string, bool $expectedResult)
     {
+        echo strlen($string);
```

Comment:
Was this checked in on accident?

Reply:
Yeah I removed it but forgot to push the update.